rails/README.md

```diff
@@ -34,10 +34,52 @@ A guide for building great Rails apps.
 
 ## Migrations
 
+Any time there is need to change the schema or data stored in the application
+database with a given group of work, write a [strong migration].
```

Comment:
Are the brackets for a link that hasn't been added yet?

Reply:
It's a reference-style link to the `[strong migration]` URL at the very bottom of the README, the end result is the same as a normal link

Reply:
Strong migrations are not related with data updates? Should we drop the `or data stored in` bit?